### PR TITLE
Gal 352/update frontend api fetch

### PR DESCRIPTION
**Pull request by zainm97**
## Description
https://nhsd-jira.digital.nhs.uk/browse/GAL-209
https://nhsd-jira.digital.nhs.uk/browse/GAL-352

Adding put and get calls to backend

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by Amar-NHS — DISMISSED** [review]

File: src/app/views/invitation_planning/InvitationPlanning.js
Comment (Amar-NHS) [inline src/app/views/invitation_planning/InvitationPlanning.js:1]: could you please brief me what was the intention behind this changes? I understood it to add the API calls to fetch data?

**Review by Amar-NHS — DISMISSED** [review]

**Review by LaithM96 — APPROVED** [review]
All checks passed, code looks good, all the ac's met the criteria from the code and the files are correct, happy to approve it. Well done, however there are some conflicts that needs to be resolved.

File: src/app/page.jsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ export default function Root() {
         <Header />
         {/* <Content /> */}
         {/* <ClinicSummary /> */}
-        {/* <InvitationPlanning /> */}
-        <ClinicInformation />
```
[the remark this answers is not in the document]
Comment (zainm97) [reply src/app/page.jsx]: Resolved

**Review by Amar-NHS — APPROVED** [review]
Looks good. Only thing would like to point out is parametrisation of api id/endpoints as it seems like will have to update it quite a few times during development.

**Review by Abdul-NHS — DISMISSED** [review]

**Review by LaithM96 — DISMISSED** [review]
all good

**Review by LaithM96 — APPROVED** [review]
happy to approve it all good.

**Review by Abdul-NHS — APPROVED** [review]

**Review by LaithM96 — APPROVED** [review]
all good, meets the ticket criteria

**Pull request merged**